### DISCUSSION TITLE
Icône alternée

### DISCUSSION THREAD

**Discussion by cinematographes** [witsa/synapps#680] [Q&A]
Bonjour,
Je sais pas si sais possible, je voulais que texte de mon bouton navigation se souligne au passage la souris, avant sur wit l'icône alternée permettait de faire. Je mettais une image avec le texte souligné et une autre avec le texte tout seul.
merci d'avance

**Comment by manufitoussiwit** [comment]
Bonjour,

Le mode "Liens" du bouton de navigation pourrait peut-être vous intéresser : [Bouton : mode](https://witsa.github.io/synapps/concepts/actor-types/input-button.html#mode)

**Reply by cinematographes** [reply]
Bonjour merci pour l'aide apporté 
Malheureusement je n'arrive pas à modifier le mode, je pense pouvoir mettre image et même sa j'y arrive pas et encore.

**Reply by cinematographes** [reply]
Je viens de mettre les image mes j'arrive que a en mettre une et donc j'arrive pas a faire le côté bouton alternée

**Reply by manufitoussiwit** [reply]
Si vous voulez alterner une image, essayez plutôt l'acteur [image interrupteur](https://witsa.github.io/synapps/concepts/actor-types/input-switch-image.html)

**Reply by cinematographes** [reply]
Merci je vais regarde ça mais le côté événement je le comprends pas trop mais je vais mi penché merci beaucoup de l'aide apporte.

**Reply by manufitoussiwit** [reply]
Avec un acteur bouton de navigation en mode Liens, ça marche plutot bien :

![link](https://github.com/witsa/synapps/assets/35595723/976fd439-9bd1-4937-90bf-e423e1382f72)

```
SYNAPPS-STUDIO-ACTOR|{"type":"input/nav-button","key":"nav-button1","properties":{"content":"Bouton de navigation","horizontalAlignment":"middle","verticalAlignment":"middle","fontSize":"30px","mode":"link"}}
```

[Pour copier/coller un acteur](https://witsa.github.io/synapps/tutorials/copy-paste-actor-scene-composite.html#copiercoller-un-acteur)

**Reply by cinematographes** [reply]
Merci je vais regarde sa mardi merci 😊
